src/app/lib/utilities/ichefURL/index.js

```diff
@@ -2,28 +2,27 @@
 const WEBP_ORIGIN_CODES = ['cpsdevpb', 'cpsprodpb'];
 
 const buildPlaceholderSrc = (src, resolution) => {
-  const parts = src.split('/');
-  const [domain, media, imgService, width, ...extraParts] = parts;
-  const definedWidth = width.replace('$width', resolution);
+  // return standard img url pips
+  if (src.includes('urn:')) return src;
+  const noProtocolSrc = src.replace('https://', '');
+  const parts = noProtocolSrc.split('/');
```

Comment:
what is parts? urlParts?

Reply:
Yes, I changed the pre-existing variable name to be more specific